PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/PictureEffectsFormat.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Drawing;
+using Microsoft.Office.Core;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Utils;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+using Shapes = Microsoft.Office.Interop.PowerPoint.Shapes;
+
+namespace PowerPointLabs.SyncLab.ObjectFormats
+{
+    class PictureEffectsFormat
+    {
+        public static bool CanCopy(Shape formatShape)
+        {
+            try
+            {
+                return formatShape.Fill.PictureEffects.Count > 0;
+            } 
+            catch
+            {
+                return false;
+            }
+        }
+
+        public static void SyncFormat(Shape formatShape, Shape newShape)
+        {
+            if (Sync(formatShape, newShape))
```

Comment:
Is there a mistake here? Why log error when sync returns true?

Reply:
Good catch there. I adopted the code from FontFormat which did the same. Missed out the fact that it should have been `!Sync(..)` instead.